lib/driver.js

```diff
@@ -910,31 +910,32 @@ class XCUITestDriver extends BaseDriver {
       args.push('-AppleLocale', this.opts.locale);
     }
 
-    let desired = {
-      desiredCapabilities: {
-        bundleId: this.opts.autoLaunch === false ? undefined : bundleId,
-        arguments: args,
-        environment: env,
-        eventloopIdleDelaySec,
-        shouldWaitForQuiescence,
-        shouldUseTestManagerForVisibilityDetection,
-        maxTypingFrequency,
-        shouldUseSingletonTestManager,
-      }
+    const wdaCaps = {
+      bundleId: this.opts.autoLaunch === false ? undefined : bundleId,
+      arguments: args,
+      environment: env,
+      eventloopIdleDelaySec,
+      shouldWaitForQuiescence,
+      shouldUseTestManagerForVisibilityDetection,
+      maxTypingFrequency,
+      shouldUseSingletonTestManager,
     };
     if (util.hasValue(this.opts.shouldUseCompactResponses)) {
-      desired.desiredCapabilities.shouldUseCompactResponses = this.opts.shouldUseCompactResponses;
+      wdaCaps.shouldUseCompactResponses = this.opts.shouldUseCompactResponses;
     }
     if (util.hasValue(this.opts.elementResponseFields)) {
-      desired.desiredCapabilities.elementResponseFields = this.opts.elementResponseFields;
+      wdaCaps.elementResponseFields = this.opts.elementResponseFields;
     }
     if (this.opts.autoAcceptAlerts) {
-      desired.desiredCapabilities.defaultAlertAction = 'accept';
+      wdaCaps.defaultAlertAction = 'accept';
     } else if (this.opts.autoDismissAlerts) {
-      desired.desiredCapabilities.defaultAlertAction = 'dismiss';
+      wdaCaps.defaultAlertAction = 'dismiss';
     }
 
-    await this.proxyCommand('/session', 'POST', desired);
+    await this.proxyCommand('/session', 'POST', {
+      firstMatch: [wdaCaps],
```

Comment:
should be `{capabilities: { firstMatch: [wdaCaps], alwaysMatch: {}, }}`?

Reply:
it will be after https://github.com/appium/WebDriverAgent/pull/203 is published and picked up